.github/workflows/open-mysql-db.yml

```diff
@@ -0,0 +1,23 @@
+name: OpenM(ysq)LDB_CICD
+
+on:
+  push:
```

Comment:
use the `paths` attribute, only trigger the job if PR changes relevant files

Reply:
DONE